app/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,7 @@
     <uses-permission android:name="android.permission.REQUEST_INSTALL_PACKAGES"/>
 
     <!-- Requested by GeckoView but not needed in VR -->
-    <uses-permission android:name="android.permission.WAKE_LOCK" tools:node="remove"/>
```

Comment:
Unfortunatelly we cannot do this for Meta devices as the store does not allow us to request WAKE_LOCK.

What you could do is to leave the permission request there and add the remove to the oculus manifests (also update the permission for Pico devices)

Reply:
Ah Okay, I will drop the commit https://github.com/Igalia/wolvic/pull/917/commits/90c81244045f56e1d95c79d5325c306b8c331887 in this case

I had some crashes when playing videos with Maven Geckoview because of this. Now I know the story behind this.